packages/ui-components/src/components/Chart/Table/getChartTableData.js

```diff
@@ -78,14 +78,11 @@ const processData = ({ data, chartType }) => {
   return data;
 };
 
-export const useChartTable = viewContent => {
+export const getChartTableData = viewContent => {
```

Comment:
@chris-bes - sorry one thing I missed in my review is that when you change the name of this function to `getChartTableData` it will throw an error about using a `hook` inside a regular function. [Hooks need to be called inside React functional components or custom hooks ](https://reactjs.org/docs/hooks-rules.html). I think we need to keep the `useMemos` for preparing the data for `react-table` so probably best to just rename this back to `useChartTable `